frontend/apps/core-app/src/navigators/recipients.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+
+import { RecipientListScreen } from '../screens/RecipientListScreen';
+import { RecipientProfileScreen } from '../screens/RecipientProfileScreen';
+
+const Stack = createStackNavigator();
+
+export const RecipientNavigator: React.FC = () => {
+  return (
+    <Stack.Navigator initialRouteName="RecipientList">
+      <Stack.Screen
+        name="RecipientList"
```

Comment:
These name strings are used in a lot of places. They should be defined as constants somewhere.

Reply:
oh yeah, thought of doing that, forgot :+1: